hs_core/templates/pages/collaborate.html

```diff
@@ -89,7 +89,7 @@ <h4 class="modal-title">Members</h4>
                                                             <tbody>
                                                             {% for u in group.gaccess.members %}
                                                                 <tr id="row-id-{{ u.pk }}"
-                                                                    {% if owners|length == 1 or not is_owner_user %}class="hide-actions"{% endif %}>
+                                                                    {% if owners|length == 1 or not self_access_level == 'owner' %}class="hide-actions"{% endif %}>
```

Comment:
is it best to be using classes, instead of v-if?

Reply:
Are there any clear benefits to v-if? I'm concerned we're using two different tools to do the same thing.  I'd prefer not to do both unless there is a clear advantage to v-if.

Reply:
v-if is simpler and better because it removes the element from the DOM if the conditional is met (not just hiding it). Vue handles all of it and will add it back if the conditional dynamically changes.

Reply:
If memory serves, we are generally sometimes doing conditional rendering, using Django template language and previously sometimes with JavaScript/jQuery (and now Vue). Ideally, we'll start to lean towards Vue and have Django conditional rendering of DOM/HTML only in cases where Vue doesn't make sense or literally can't do it (access to Python scope impossible or laborious, hard to read code)